tests/test_schema.py

```diff
@@ -3023,13 +3047,34 @@ def test_create_new_unique_identity(self):
 
         uidentity_db = UniqueIdentity.objects.get(uuid='03877f31261a6d1a1b3971d240e628259364b8ac')
         identities_db = uidentity_db.identities.all()
-        self.assertEqual(len(identities_db), 1)
+        self.assertEqual(len(identities_db), 2)
 
         identity_db = identities_db[0]
         self.assertEqual(identity_db.id, '03877f31261a6d1a1b3971d240e628259364b8ac')
         self.assertEqual(identity_db.name, None)
         self.assertEqual(identity_db.email, 'jdoe@example.com')
 
+        identity_db = identities_db[1]
+        self.assertEqual(identity_db.id, '0880dc4e621877e8520cef1747d139dd4f9f110e')
+        self.assertEqual(identity_db.name, 'J. Smith')
+        self.assertEqual(identity_db.email, 'jsmith@example.org')
+
+    def test_from_id_match_uuid(self):
```

Comment:
You can change the name of this tests to `test_from_id_is_unique_identity` or similar.

Reply:
Done!